src/store/index.tsx

```diff
@@ -64,6 +68,16 @@ export const DerivProvider = ({ children }: DerivProviderProps) => {
         }
     }, [website_status])
 
+    useEffect(() => {
+        const branding = () => {
+            const host_name = location.hostname
+            const domain_name = getDomainName()
+            return branding_domains.some((item) => item === host_name || item === domain_name)
+        }
+
+        setHideBranding(branding)
```

Comment:
It's not recommended to store a function in a state. Can we just have it as a normal function?

Reply:
okay